extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -82,40 +82,49 @@ export class DbPanel extends DisposableObject {
   }
 
   private async addNewRemoteDatabase(): Promise<void> {
-    const quickPickItems = [
-      {
-        label: "$(repo) From a GitHub repository",
-        detail: "Add a remote repository from GitHub",
-        alwaysShow: true,
-        kind: "repo",
-      },
-      {
-        label: "$(organization) All repositories of a GitHub org or owner",
-        detail:
-          "Add a remote list of repositories from a GitHub organization/owner",
-        alwaysShow: true,
-        kind: "owner",
-      },
-    ];
-    const databaseKind =
-      await window.showQuickPick<RemoteDatabaseQuickPickItem>(quickPickItems, {
-        title: "Add a remote repository",
-        placeHolder: "Select an option",
-        ignoreFocusOut: true,
-      });
-    if (!databaseKind) {
-      // We don't need to display a warning pop-up in this case, since the user just escaped out of the operation.
-      // We set 'true' to make this a silent exception.
-      throw new UserCancellationException("No repository selected", true);
-    }
-    if (databaseKind.kind === "repo") {
-      await this.addNewRemoteRepo();
-    } else if (databaseKind.kind === "owner") {
-      await this.addNewRemoteOwner();
+    const highlightedItem = await this.getHighlightedDbItem();
+
+    if (highlightedItem?.kind === DbItemKind.RemoteUserDefinedList) {
+      await this.addNewRemoteRepo(highlightedItem.listName);
+    } else {
```

Comment:
From the (internal) issue: 
> If a list or item inside a list is highlighted, add it to that list

So I think that means something like this? (i.e. also checking if the highlighted item is inside a list)
```suggestion
    if (highlightedItem?.kind === DbItemKind.RemoteUserDefinedList) {
      await this.addNewRemoteRepo(highlightedItem.listName);
    } else if (
      highlightedItem?.kind === DbItemKind.RemoteRepo &&
      highlightedItem.parentListName
    ) {
      await this.addNewRemoteRepo(highlightedItem.parentListName);
    } else {
```

Reply:
Oh yes, didn't think about that! Will add the case!